src/sessions/persistent-session.ts

```diff
@@ -0,0 +1,307 @@
+/**
+ * @fileoverview A Persistent Box API Session.
+ */
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+import { Promise } from 'bluebird';
+import assert = require('assert');
+import errors = require('../util/errors');
+import httpStatusCodes = require('http-status');
```

Comment:
These imports all work the same if they're formatted like `import assert from 'assert';`, right? Wondering if we should settle on that format for consistency

Reply:
Yes, once all dependencies are translated to TypeScript (and using `export` syntax) it's okay we can stick to `import ... from` syntax. I've made updates to the PR. Thanks!